esp/services/esdl_svc_engine/esdl_svc_custom.cpp

```diff
@@ -357,12 +358,16 @@ void CEsdlCustomTransform::processTransform(IEspContext * context, StringBuffer
             }
         }
 
-        Owned<IPropertyTree> thereq = createPTreeFromXMLString(request.str());
+        Owned<IPropertyTree> theroot = createPTreeFromXMLString(request.str());
+        StringBuffer xpath = m_target.str();
+        if (!xpath.length())
+            xpath.setf("soap:Body/%s/%s", mthdef.queryMethodName(), mthdef.queryRequestType()); //This default gives us backward compatibility with only being able to write to the actual request
```

Comment:
Current roxie requests use an xpath like `soap:Body/MyRoxieService/MyESPMethodRequest`. This looks like it will generate `soap:Body/MyESPMethod/MyESPMethodRequest`. Will this work when MyRoxieService and MyESPMethod do not have the same value?

Reply:
Good point that may have just worked for my deployment.  It can always be over riden, but let me try to find where I can get the roxie service name from.